VocaDbWeb/Program.cs

```diff
@@ -17,6 +18,17 @@
 
 	startup.ConfigureServices(builder.Services);
 
+	builder.Services.AddRateLimiter(options =>
+	{
+		options.RejectionStatusCode = StatusCodes.Status429TooManyRequests;
+		options
+			.AddSlidingWindowLimiter(policyName: "details", options =>
+			{
+				options.Window = TimeSpan.FromMinutes(1);
+				options.PermitLimit = 40;
+				options.SegmentsPerWindow = 1;

```

Comment:
This code doesn't match the PR description.

> 40 req/sec 

```cs
TimeSpan.FromMinutes(1)
```

> max. 4 concurrent requests 

Nowhere to be seen.

Reply:
That's why it's still a draft PR. (You've probably also noticed that the rate limit is currently global and partitioned by the IP address of the user agent)